### PR TITLE
Fix runtime error, update tests

### PR DESCRIPTION
**Pull request by judd-stytch**
Sorry for combining multiple issues in one single commit PR. Also, fair warning I have never written Python for more than toy scripts.

1.  I noticed I was getting a runtime error when running a python demo app with the Stytch python lib. It seems to be due to the type hint not having quotes around it (see above warning about my Python knowledge).
2. The unit tests were failing due to a version bump
3. I deleted the integration test since it was failing due to an uncaught warning about the test env. After discussion with @grace-stytch and @julianna-stytch we decided to delete (client lib probably isn't a great place for an integration test to live anyways).

### CODE REVIEW

File: stytch/api/users.py [hunk 1]
```diff
@@ -60,8 +60,8 @@ def delete(self, user_id: str):
     def update(
         self,
         user_id: str,
-        emails: Optional[list[str]] = None,
-        phone_numbers: Optional[list[str]] = None,
+        emails: 'Optional[list[str]]' = None,
+        phone_numbers: 'Optional[list[str]]' = None,
```
Comment (jeremy-stytch) [inline stytch/api/users.py:64]: Nice find! `mypy` doesn't the stringification, but this should work instead: 
```suggestion
        emails: Optional[List[str]] = None,
        phone_numbers: Optional[List[str]] = None,
```
with an extra import up top:
```diff
-from typing import Dict, Optional
+from typing import Dict, List, Optional
```

**Review by jeremy-stytch — COMMENTED** [review]
Assigning @nikhil-stytch who's been handling this library recently

File: stytch/api/users.py (hunk 1 above)
Comment (judd-stytch) [reply stytch/api/users.py:64]: Ah, yup, that makes sense! Thanks!

**Review by jeremy-stytch — APPROVED** [review]
LGTM

**Pull request merged**